vignettes/depends_showcase.Rmd

```diff
@@ -96,7 +96,10 @@ project_file_network <- transform_dependencies_to_network(project_dependencies_s
                                                    weight_types = keep_dependencies_type)
 
 kable(head(project_dependencies[["edgelist"]],10))
+
 ```
+
+
```

Comment:
I think this is a collateral change, but since it doesn't touch the code it's alright.